### DISCUSSION TITLE
Einige Fragen als Neuling. Betrieb ohne Batterie.

### DISCUSSION THREAD

**Discussion by firedp** [hoylabs/OpenDTU-OnBattery#821] [General]
Hallo, Frohe Ostern, Glückwunsch und meinen großen Dank für Euer Pr% ojekt.
Als Laie habe ich es geschafft mit Hichi-Tasmota und einer vorhandenen openDTU es auf on Battery zum Laufen zu bringen.
Trotzdem habe ich Fragen zu Einstellungen und Euren Empfehlungen zu Einstellungen. Derzeit habe ich 2 hm-600 mit (nur) 3 Modulen im Betrieb und in die DTU eingerichtet. Ein hm läuft ungeregelt.

Zu Wechselrichter: Erreichbarkeit Schwellenwert habe ich intuitiv auf 3 gestellt, Nulle Laufzeit und Tagesertrag auf off und Tagesertragskorrektur auf on. Wie habt Ihr das und warum?

Was ist VE.Direct?

Stromzähler Einstellungen
Ein für mich einfacher Weg war HTTP(S)+JSON, da es einfach und bildliche Erklärungen im Netz gibt, wäre was anderes besser und schneller?
Was ist Individuelle HTTP requests pro Phase? Bei mir off.
Timeout steht auf 1000ms, was bewirkt es und was ist ein guter Wert?

Dynamic Power Limiter Einstellungen 
Angestrebter Netzbezug. Ich verstehe es so, bei meinem Wert -(minus) 307W erlaube ich bis 307W ins Netz einzuspeisen. Für mich OK, denn ich habe ja ein angemeldetes BKW und bis jetzt auch Überschuss verschenkt. Wie haltet Ihr das?
Hysterese habe ich mal auf 25W gestellt, ist es zu viel oder wenig?
Oberes Leistungslimit auf 600W, schließlich brauche ich die vielleicht mal im Optimalfall das untere habe ich mal auf 16W gestellt, ob das zu niedrig ist? Eigentlich könnte ich das untere auch höher (z.B. 50W) einstellen, selbst wenn ich 2x 600W produzieren würde, käme ich bei einer Grundlast von 100W im Haus mit eurer Software auch nicht auf über 500W Netzeinspeisung, richtig?

DTU Einstellungen, da habe ich 10 Sekunden Abfrage, trotzdem passiert zwischendurch das eine Aktualisierung 14-19 Sekunden dauert. Normal? Was habt ihr eingestellt?

Sonst etwas was ihr empfehlen könnt?

Zum Schluss der Lesekopf. In der Anleitung stand, man solle kein Update machen. Aber kann man und macht es Sinn dort etwas anzupassen? Anpassungen wie oft ausgelesen und Werte bereitgestellt werden? Oder andere Dinge?

Vielen Dank für Eure Zeit und Verständnis. Vielleicht haben andere Laien ähnliche Fragen, auch wenn sie wie ich das System zum Laufen gebracht haben.

Noch zur Speicherinformation. Was ist Sketch und ist es OK das es bei 99% steht?

**Comment by Manos1966** [comment]
Frohe Ostern @firedp 

>Als Laie habe ich es geschafft mit Hichi-Tasmota und einer vorhandenen openDTU es auf on Battery zum Laufen zu bringen.

👍 


> Zu Wechselrichter: Erreichbarkeit Schwellenwert habe ich intuitiv auf 3 gestellt, Nulle Laufzeit und Tagesertrag auf off und Tagesertragskorrektur auf on. Wie habt Ihr das und warum?

Über Geschmack lässt sich streiten, die meisten Einstellungen dienen den persoenlichen Nutzen jeweiliger Anwender. 
Wenn du deine Maus ueber das "info" Zeichen bringst, kannst du eine Beschreibung der Funkion lesen.
Da du eine relativ "Einfache" Installation hast, ist es normal, dass du den Nutzen nicht sofort erkennen kannst, brauchst du auch zur Zeit noch nicht.

![Capture](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/042b7b39-04c9-4713-b33b-316d262eaa8f)

> Was ist VE.Direct?

https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/VE.Direct-Interface

> Ein für mich einfacher Weg war HTTP(S)+JSON, da es einfach und bildliche Erklärungen im Netz gibt, wäre was anderes besser und schneller?

Nein

> Was ist Individuelle HTTP requests pro Phase? Bei mir off.

Strom im Haus hat Drei Phasen.

> Timeout steht auf 1000ms, was bewirkt es und was ist ein guter Wert?

Ja

> Dynamic Power Limiter Einstellungen

https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Dynamic-Power-Limiter

> Angestrebter Netzbezug. Ich verstehe es so, bei meinem Wert -(minus) 307W erlaube ich bis 307W ins Netz einzuspeisen. Für mich OK, denn ich habe ja ein angemeldetes BKW und bis jetzt auch Überschuss verschenkt. Wie haltet Ihr das?
👍 

> Hysterese habe ich mal auf 25W gestellt, ist es zu viel oder wenig? 
👍 

> das untere habe ich mal auf 16W gestellt, ob das zu niedrig ist? 

Stelle es lieber auf 30W

> Eigentlich könnte ich das untere auch höher (z.B. 50W) einstellen, selbst wenn ich 2x 600W produzieren würde, käme ich bei einer Grundlast von 100W im Haus mit eurer Software auch nicht auf über 500W Netzeinspeisung, richtig?

Richtig


> DTU Einstellungen, da habe ich 10 Sekunden Abfrage, trotzdem passiert zwischendurch das eine Aktualisierung 14-19 Sekunden dauert. Normal? Was habt ihr eingestellt?

Na ja, bei 19 Sek. es scheint als ob Kommunikations-Probleme existieren. Es haengt vom Abstand zwischen deine DTU und deine HM-600, von deiner NRF24 Platine (hat sie eine externe Antenne?) und von der NRF24 Sendeinstellung:
![Capture](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/d24a17e1-1b9f-4195-a6d4-7dd2d64626d7)


> Zum Schluss der Lesekopf. In der Anleitung stand, **man solle kein Update machen**. 

Spiel nicht mit dem Feuer....  😄 

> Noch zur Speicherinformation. Was ist Sketch und ist es OK das es bei 99% steht?

Es ist OK

**Comment by spcqike** [comment]
Dir auch frohe Ostern 

> Nulle Laufzeit und Tagesertrag auf off und Tagesertragskorrektur auf on. Wie habt Ihr das und warum?

genauso. Alles andere macht mir einen System ohne Batterie keinen Sinn. Das bräuchte man nur bei Batteriesystemen, da die wechselrichter dort dauerhaft laufen und nicht aus gehen (anders als bei PV. Wenn die Sonne weg ist, geht der wechselrichter aus. Sie speisen sich durch die DC Seite) 

> Was ist VE.Direct?

die Victron Schnittstelle für laderegler und/oder smartshunts. 

> wäre was anderes besser und schneller?

definiere besser und schneller. 

Ressourcensparsamer auf dem esp wäre wohl MQTT. Schneller, im Sinne von der Dynamic power limiter task läuft schneller durch, wäre auch MQTT. 
Schneller, im Sinne von die leistungswerte des Smart Meters sind näher an den Werten des wechselrichters, ist meiner Meinung nach https+json (da in dem Fall das powermeter direkt vom power limiter gezielt abgefragt wird)

> Timeout steht auf 1000ms, was bewirkt es und was ist ein guter Wert?

die Zeit, die der esp auf eine Antwort des powermeters wartet, bevor er es als „nicht verfügbar“ interpretiert. So lange er auf eine Antwort wartet, steht die openDTU quasi und wartet eben nur. 

> Ich verstehe es so, bei meinem Wert -(minus) 307W erlaube ich bis 307W ins Netz einzuspeisen.

Nein. Du strebst damit -307W als Ziel an. Der limiter wird versuchen, so viel zu produzieren, bis 307W eingespeist werden. 
wenn du das ok findest, kannst du den limiter auch aus lassen und den WR auf 100% laufen lassen. Das schließt Fehlerquellen aus. (Bspw den Ausfall des powermeters, durch Netzwerk oder Netzteil oder was auch immer) 
Ich habe ein angestrebten Bezug von -20W. Damit mein Bedarf gedeckt wird Aber ich das Netz nicht zusätzlich „belaste“. (Ja, darüber lässt sich streiten. Aber wenn der Strom am Wochenende zur Mittagszeit auf der Börse kostenlos ist, müssen Kraftwerke und Kapazitäten reduziert werden, sonst rutscht der Preis ins negative.)

> Hysterese habe ich mal auf 25W gestellt, ist es zu viel oder wenig?

Ich hab 10. aber passt. Denke ich. 

> untere habe ich mal auf 16W gestellt

Stell das untere Limit lieber auf einen höheren Wert. Wenn du sie kennst, auf deine bekannte Grundlast. Ich habe 200W eingestellt. 
Damit speist du mindestens 200W ein, wenn die Sonne es erlaubt. Auch dann, wenn bspw das powermeter ausfällt und du nicht weißt, wie hoch dein Verbrauch genau ist, wird auf 200W geregelt. 
wie gesagt: ein sinnvoller wert als Grundlast für dich.

**Reply by Manos1966** [reply]
ich gehe davon aus, es war die Zeit-Umstellung @spcqike 
😄 😄 😄 😄 😄 😄

**Comment by firedp** [comment]
Vielen Dank, ich bin immer froh zu verstehen und es nicht einfach hinzunehmen und laufen zu lassen.
Wer nicht fragt bleibt dumm :-)
